crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/builder/MaatCourtDataAssessmentBuilder.java

```diff
@@ -85,19 +86,25 @@ private MaatApiCreateAssessment buildCreate(MeansAssessmentRequestDTO requestDTO
 
     private MaatApiUpdateAssessment buildUpdate(MeansAssessmentDTO assessment) {
         MeansAssessmentRequestDTO meansAssessment = assessment.getMeansAssessment();
-        return new MaatApiUpdateAssessment()
-                .withFullAscrId(assessment.getAssessmentCriteria().getId())
-                .withFullAssessmentDate(meansAssessment.getFullAssessmentDate())
-                .withFullResult(ofNullable(assessment.getFullAssessmentResult())
-                        .map(FullAssessmentResult::getResult).orElse(null))
-                .withFullResultReason(ofNullable(assessment.getFullAssessmentResult())
-                        .map(FullAssessmentResult::getReason).orElse(null))
-                .withFullAssessmentNotes(meansAssessment.getFullAssessmentNotes())
-                .withFullAdjustedLivingAllowance(assessment.getAdjustedLivingAllowance())
-                .withFullTotalAnnualDisposableIncome(assessment.getTotalAnnualDisposableIncome())
-                .withFullOtherHousingNote(meansAssessment.getOtherHousingNote())
-                .withFullTotalAggregatedExpenses(assessment.getTotalAggregatedExpense())
-                .withUserModified(meansAssessment.getUserSession().getUserName())
-                .withFinancialAssessmentId(meansAssessment.getFinancialAssessmentId());
+
+        MaatApiUpdateAssessment updateAssessment = new MaatApiUpdateAssessment();
+
+        if (AssessmentType.FULL.equals(assessment.getMeansAssessment().getAssessmentType())) {
```

Comment:
assessment.getMeansAssessment() is already defined on line number 88, can we use meansAssessment?